deepcell_label/blueprints.py

```diff
@@ -94,24 +93,13 @@ def add_raw(token):
     return {'numChannels': project.num_channels}
 
 
-@bp.route('/api/labeled/<token>/<int:feature>/<int:frame>')
-def labeled(token, feature, frame):
-    project = Project.get(token)
-    if not project:
-        return abort(404, description=f'project {token} not found')
-    png = project.get_labeled_png(feature, frame)
-    etag = hashlib.md5(png.getbuffer()).hexdigest()
-    response = send_file(png, mimetype='image/png', cache_timeout=0, etag=etag)
-    return response
-
-
 @bp.route('/api/array/<token>/<int:feature>/<int:frame>')
 def array(token, feature, frame):
     """ """
     project = Project.get(token)
     if not project:
         return jsonify({'error': f'project {token} not found'}), 404
-    labeled_array = project.get_labeled_array(feature, frame)
+    labeled_array = project.get_labeled_array(feature, frame).flatten()
```

Comment:
Why is this change required?

Reply:
I made this change to work better with GPU.js, but I'm not 100% sure it's needed... let me experiment a bit and see if it's kosher to pass a 2D array to GPU.js, and if it is, I'll prefer that and send the unflattened arrays.

Reply:
Got it to work, so it's switched back now to sending 2D arrays instead of flattening into a 1D array